code42.json

```diff
@@ -1924,6 +1924,964 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "hunt file",
```

Comment:
these names (`hunt file` and `run query`) are based off the name from the old app. I remember Phantom specifically saying that they try to name certain types of actions with the same action name because this makes them get grouped together in the UI (i.e. you could pick "hunt file" and then choose which app you want to hunt the file in).